src/main/java/com/exasol/adapter/dialects/oracle/OracleColumnMetadataReader.java

```diff
@@ -33,31 +31,27 @@
      * @param identifierConverter converter between source and Exasol identifiers
      */
     public OracleColumnMetadataReader(final Connection connection, final AdapterProperties properties,
-            final IdentifierConverter identifierConverter) {
+                                      final IdentifierConverter identifierConverter) {
         super(connection, properties, identifierConverter);
     }
 
     @Override
     public DataType mapJdbcType(final JdbcTypeDescription jdbcTypeDescription) {
         switch (jdbcTypeDescription.getJdbcType()) {
-        case Types.DECIMAL:
-        case Types.NUMERIC:
-            return mapNumericType(jdbcTypeDescription);
-        case ORACLE_TIMESTAMP_WITH_TIME_ZONE:
-        case ORACLE_TIMESTAMP_WITH_LOCAL_TIME_ZONE:
-            return DataType.createTimestamp(false);
-        case Types.NCLOB:
-        case ORACLE_CLOB:
-        case INTERVAL_YEAR_TO_MONTH:
-        case INTERVAL_DAY_TO_SECOND:
-        case ORACLE_BINARY_FLOAT:
-        case ORACLE_BINARY_DOUBLE:
-        case Types.ROWID:
-            return DataType.createMaximumSizeVarChar(DataType.ExaCharset.UTF8);
-        case Types.BLOB:
-            return mapBlobType();
-        default:
-            return super.mapJdbcType(jdbcTypeDescription);
+            case Types.DECIMAL:
+            case Types.NUMERIC:
+                return mapNumericType(jdbcTypeDescription);
+            case ORACLE_TIMESTAMP_WITH_TIME_ZONE:
+            case ORACLE_TIMESTAMP_WITH_LOCAL_TIME_ZONE:
+                return DataType.createTimestamp(false);
+            case INTERVAL_YEAR_TO_MONTH:
+            case INTERVAL_DAY_TO_SECOND:
+            case ORACLE_BINARY_FLOAT:
+            case ORACLE_BINARY_DOUBLE:
+            case Types.ROWID:
```

Comment:
According to the data mapping table, this data type is not supported. Check it in the test. If there is no test, create one. If it's not supported, remove it from here too